tests/NerlnetFullFlowTest.sh

```diff
@@ -13,7 +13,8 @@ NERLNET_CONFIG_INPUT_DATA_DIR=$NERLNET_CONFIG_DIR/inputDataDir.nerlconfig
 NERLNET_CONFIG_INPUT_DATA_DIR_BACKUP=$NERLNET_CONFIG_DIR/inputDataDir.nerlconfig.bac
 
 TEST_INPUT_JSONS_FILES_DIR="$TESTS_PATH/inputJsonsFiles"
-export TESTS_BASELINE="$TEST_INPUT_JSONS_FILES_DIR/accuracy_stats_synt_1d_2c_4r_4w.json"
+export TEST_BASELINE_ACC_STATS="$TEST_INPUT_JSONS_FILES_DIR/accuracy_stats_synt_1d_2c_4r_4w.json"
```

Comment:
@NoaShapira8 @ohad123 
It's F1 score not accuracy.
I merged it but fix this typo in next PR